### ISSUE TITLE
Make new patch release 2.1.2 of OMSimulator

### ISSUE THREAD

**Issue by casella** [OpenModelica/OMSimulator#1274]
We fixed several issues in OMSimulator since the last [patch release  2.1.2](https://github.com/OpenModelica/OMSimulator/releases/tag/v2.1.1).

There are some fixes in om_maintenance/v1.21 that were made by @adrpo for OpenModelica 1.21.0. There are also a lot of fixes on the master branch related to the use of CMAKE, library updates, and some more recent fixes by @arun3688 like #1273, that can really make a difference in terms of end-user experience. 

I believe it is now definitely time to make a new patch release 2.1.2 of OMSimulator that includes all this stuff, just in case somebody wants to use OMSimulator independently of OpenModelica.

I would also make sure that all these changes are available in OpenModelica 1.22.1.

@lochel would you like to comment on that?

**Comment by lochel** [comment]
Sound like a good idea 👍 

Besdies that, we have the nightly builds [1] and also the pip releases [2]. The latest nightly build is version v2.1.1.post203 and the latest pip release is using version v2.1.1.post190.

[1] https://build.openmodelica.org/omsimulator/nightly/
[2] https://pypi.org/project/OMSimulator/

**Comment by lochel** [comment]
Yes, that's not a problem. However, the nightly builds are currently broken. That should be fixed as soon as possible.

**Comment by arun3688** [comment]
@lochel you mean the nightly builds are broken with related to OMSimulator update ?

**Comment by lochel** [comment]
I see that we created the patch release v2.1.2 in March this year. However, the artifacts were broken, and no corresponding pip package was generated or published at that time.

I have now created a new patch release, v2.1.3, with all artifacts working correctly. Additionally, a new pip package for this version has been successfully generated and published. Thanks to @adrpo, the entire Jenkins pipeline is now much more stable.

Furthermore, I have updated the OMSimulator submodule in OpenModelica to reference this new release.
